lib/openshift/node.rb

```diff
@@ -79,6 +79,12 @@ def is_worker?(user: nil, cached: true, quiet: false)
       return ! res.nil?
     end
 
+    def is_master?(user: nil, cached: true, quiet: false)
```

Comment:
@xiuwang there's already a `is_worker?` method (https://github.com/openshift/verification-tests/blob/master/lib/openshift/node.rb#L76).  @akostadinov, do you think we can reuse that method?  This is with the assumption that there will only be two types of host (if not a worker, then it's a master)...